src/ui/widgets/alignwidget.h

```diff
@@ -50,6 +50,7 @@ class UI_EXPORT AlignWidget : public QWidget
 
     QComboBox *mAlignPivot;
     QComboBox *mRelativeTo;
+    QComboBox *mRelativeToPivot;
```

Comment:
add `, mRelativeToPivot(nullptr)` in https://github.com/friction2d/friction/blob/main/src/ui/widgets/alignwidget.cpp#L39

Reply:
OK, I will

Reply:
See my last comment, let's figure out what to do with the UI first.

Reply:
btw, why does `mRelativeToPivot` exists?

Reply:
Oh, I remember it now, it could be removed.
In my first development cycle I only implemented "relative to bounding box" and "relative canvas" and the technique I used was to hide the normal relativeTo comboBox and show the new one when "align to pivot" was selected in the other comboBox but now all options are used (Last selected ones) so this one could be removed... my fault not to realize about it before and clean it

Reply:
Do you want me to do it?